prefq/examples/imitation_preference_comparisons.py

```diff
@@ -5,27 +5,194 @@
 # Disable formatting and linters so we can keep the example as close to the original as possible for now.
 # fmt: off
 # pylint: skip-file
+
+import time
+import uuid
+import json
+import os
+import pathlib
+import tempfile
+from collections import defaultdict
+from typing import Optional, Sequence, Tuple
+
 import numpy as np
+import requests
 from imitation.algorithms import preference_comparisons
+from imitation.algorithms.preference_comparisons import (
+    SynchronousHumanGatherer,
+    write_fragment_video,
+)
+from imitation.data.types import TrajectoryWithRewPair
+from imitation.data.wrappers import RenderImageInfoWrapper
 from imitation.policies.base import FeedForward32Policy, NormalizeFeaturesExtractor
 from imitation.rewards.reward_nets import BasicRewardNet
 from imitation.rewards.reward_wrapper import RewardVecEnvWrapper
+from imitation.util import logger as imit_logger
+from imitation.util import video_wrapper
 from imitation.util.networks import RunningNorm
 from imitation.util.util import make_vec_env
 from stable_baselines3 import PPO
 from stable_baselines3.common.evaluation import evaluate_policy
 from stable_baselines3.ppo import MlpPolicy
 
+
+class PrefqGatherer(SynchronousHumanGatherer):
```

Comment:
Please add a comment that clarifies that this is a slightly modified version of the `PrefCollectGatherer` from https://github.com/HumanCompatibleAI/imitation/pull/716.